packages/contract/test/deploy.js

```diff
@@ -225,8 +224,8 @@ describe("Deployments", function() {
 
       assert(multiplier === 1.25, "Multiplier should be initialized to 1.25");
       assert(
-        multiplier * estimate > block.gasLimit,
-        "Multiplied estimate should be too high"
+        multiplier * estimate < block.gasLimit,
```

Comment:
Huh, what's the reason for this change?

Reply:
This test is/was confusing because the `it` statement says the multiplied estimate *should* be less than the `blockLimit` and we were previously asserting that it *was* higher than the `blockLimit`.

After chatting with @nicholasjpaterno about this test case, it appears the gas cost for creating the `Example` contract got much cheaper (by 2/3rds) due to a dirty storage slot bug fix in `istanbul` (which ganache now defaults to). See [EIP 2200](https://eips.ethereum.org/EIPS/eip-2200) & [EIP 2028](https://eips.ethereum.org/EIPS/eip-2028).

Thus, it made sense to reverse the sign since this appears to be in line with the original intent of this specific test.